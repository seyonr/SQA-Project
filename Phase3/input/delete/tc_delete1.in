login 
admin
12345
DE
54321
Jane Doe