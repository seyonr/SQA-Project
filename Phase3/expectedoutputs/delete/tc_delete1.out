Welcome to the banking system!
Do you want to login or exit : 
Enter login type (admin or standard): 
Enter account number: 
Login successful!

What do you want to do today?
W – Withdrawal
T - Transfer
P - Paybill
D - Deposit
C - Create
DE - Delete
DI - Disable
CH - Change Plan
L – Logout
Please enter the account number of the account to delete : 
Please enter the account name of the account to delete : 
Account deleted successfully, session terminated!
